### PR TITLE
Extensions

### PR DESCRIPTION
**Pull request by sajayantony**
Signed-off-by: Sajay Antony <sajaya@microsoft.com>

This consumes feedback from https://github.com/opencontainers/distribution-spec/pull/111#issuecomment-867251252 from @stevvooe

### CODE REVIEW

**Comment by sajayantony** [comment]
Need to include the root level extension as per the comment above. Will push and update to this. PR.

File: ext/README.md [hunk 1]
```diff
@@ -16,30 +16,37 @@ The basis of the Extension API is described in a document which should be emulat
...
+Names MAY include a version as part after the extension name. Recommendation is to avoid version 
+and use `mediaType` directly to ensure behaviour change when possible.
 
-Each extension's endpoints will be nested below its name.
+Each extension's endpoints will be nested below its name similar to the
+base extension accessible under a repository.
 
 ```HTTP
-    GET /v2/ext/0/...
+    GET /v2/{name}/_ext/
```
Comment (sajayantony) [inline ext/README.md:33]: Need to add details for root extension

**Comment by vbatts** [comment]
I did not even see this PR! Very sorry.

**Comment by vbatts** [comment]
@sajayantony i'm now wondering, which doc is most up to date? this PR or your hackmd?

**Comment by sajayantony** [comment]
Hackmd. We are good. 😊

**Pull request closed without merging**